### DISCUSSION TITLE
Where is a changelog for GitHub Copilot

### DISCUSSION THREAD

**Discussion by dzienisz** [community/community#61580] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

It would be nice to see changelog for GitHub Copilot

**Comment by Akash1134** [comment, marked as answer]
Hi @dzienisz ,

For changelog specific to VSCode bases extension: see this [changelog](https://marketplace.visualstudio.com/items/GitHub.copilot/changelog)

In your IDE, you can click on the Changelog section in the Copilot extension.

We've shared detailed updates on ships, improvements, and updates we've made in June and recently in the month of July: 
- [June Copilot updates](https://github.blog/changelog/2023-06-29-copilot-june-2023-update)
- [July 14 updates](https://github.blog/changelog/2023-07-14-github-copilot-july-14th-update)

Thanks!

**Comment by davecheney** [comment]
Duplicate of https://github.com/orgs/community/discussions/109521
